cmd/statusd/utils.go

```diff
@@ -1355,9 +1358,6 @@ func startTestNode(t *testing.T) <-chan struct{} {
 			t.Log("Node started, but we wait till it be ready")
 		}
 		if envelope.Type == signal.EventNodeReady {
-			// manually add static nodes (LES auto-discovery is not stable yet)
-			PopulateStaticPeers()
```

Comment:
Why?

Reply:
I've remove this method. I think it's unuseful.
{code}
-func PopulateStaticPeers() *C.char {
 -	err := fmt.Errorf("%v: %v", common.ErrDeprecatedMethod.Error(), "PopulateStaticPeers")
 -	return makeJSONResponse(err)
 -}
{code}

Reply:
Ok, good decision.